packages/engine/src/audio/systems/MediaSystem.ts

```diff
@@ -107,10 +110,37 @@ export class AudioEffectPlayer {
 
 globalThis.AudioEffectPlayer = AudioEffectPlayer
 
-export const MediaPrefabs = {
-  audio: 'Audio' as const,
-  video: 'Video' as const,
-  volumetric: 'Volumetric' as const
+export const MediaElements: SceneElementsRecord = {
```

Comment:
`SceneElementsRecord` and `ISceneElement` are unnecessary. we just need a list of components, which we already have in `PrefabEditors.tsx`